Dockerfile

```diff
@@ -31,6 +31,7 @@ RUN npm run build
 FROM node:lts-alpine as prod
 COPY --from=dev /app/node_modules node_modules
 COPY --from=build-prod /app/dist /
+RUN wget https://s3.amazonaws.com/rds-downloads/rds-combined-ca-bundle.pem
```

Comment:
I think it would be better to download and just commit the file into the repo rather than have it pull it each time the container is built.

Reply:
Note, happy for this to be deferred into another ticket if you don't want to resolve it now.